packages/library/src/commonMain/kotlin/io/realm/internal/RealmObjectCompanion.kt

```diff
@@ -18,11 +18,13 @@ package io.realm.internal
 
 import io.realm.interop.Table
 import kotlin.reflect.KMutableProperty1
+import kotlin.reflect.KProperty1
 
 // TODO MEDIATOR/API-INTERNAL Consider adding type parameter for the class
 @Suppress("VariableNaming")
 interface RealmObjectCompanion {
     val `$realm$fields`: List<KMutableProperty1<*, *>>?
+    val `$realm$excludedProperties`: List<KProperty1<*, *>>?
```

Comment:
Why do we need this?

Reply:
It facilitates accessing the excluded properties from a test case, and it can help to debug.